openlibrary/plugins/upstream/utils.py

```diff
@@ -1314,6 +1314,16 @@ def get_location_and_publisher(loc_pub: str) -> tuple[list[str], list[str]]:
     return ([], [loc_pub.strip(STRIP_CHARS)])
 
 
+@functools.cache
+def get_edu_domains():
+    f = open('static/edu-domains.txt')
```

Comment:
I think somewhere in scripts we may have an e.g. of abspath being used (re: whether this relpath references the file as we want)